Azure-Samples/communication-services-android-calling-hero
# Fix various intents opening up multiple/duplicate activities

## Purpose

* Fixes button presses triggering multiple times, which opens duplicate dialogues/windows

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

- Ensure rapid clicks (Best done on a real device) don't open up multiple activities or dialogues
